data/post-scripts/70-generate-lvm-device-file.ks

```diff
@@ -0,0 +1,9 @@
+# Generate LVM devices file
+# /etc/lvm/devices/system.devices
+# See: https://bugzilla.redhat.com/show_bug.cgi?id=2002550
+
+%post
+
+vgimportdevices -a
```

Comment:
I think it would be good to check the tool is there or make this non-fatal if it can't be invoked.